### PR TITLE
Change config files to kube-system which were missed in the original PR

### PR DESCRIPTION
**Pull request by satnam6502**
I missed these in the original PR #10487

### CODE REVIEW

**Comment by satnam6502** [comment]
@zmerlynn and some TLs: @dchen1107 @quinton-hoole

**Comment by marekbiskup** [comment]
cc @lhuard1A @WIZARD-CXY (who authored the addon manifests for libvirt-coreos and ubuntu)

**Comment by marekbiskup** [comment]
@a-robinson,
Shouldn't we rebuilt the Docker image after we change configuration (contrib/logging/fluentd-sidecar-es/td-agent.conf)
I think all tests passed without rebuilding the Docker image. Either this change is not necessary, or we're not testing some scenarios.

**Comment by satnam6502** [comment]
Good point: I've now created a `:1.1` image and updated the PR. Thanks.

**Comment by WIZARD-CXY** [comment]
@satnam6502 I am working on the dns update to v4 in this pr. https://github.com/GoogleCloudPlatform/kubernetes/pull/10393. This pr took too long for a merge.In this pr I changed the template file api version to v1.

**Comment by WIZARD-CXY** [comment]
Certainly you can get merged before me.I will rebase then.

**Comment by marekbiskup** [comment]
Still, I suppose we have no e2e test running for fluentd-sidecar-es. @a-robinson, could you check what is missing and report an issue? (Or, maybe we already have an issue).

**Comment by marekbiskup** [comment]
LGTM

I suggest @a-robinson for the second reviewer because he authored fluentd-sidecar-es. I'll feel safer if he reviews it.

**Comment by a-robinson** [comment]
Fluentd sidecar changes LGTM

**Comment by a-robinson** [comment]
It's not being tested because it's in contrib rather than a core system component, which I think is WAI in principle, although could be improved for this case.

**Comment by alex-mohr** [comment]
@zmerlynn presumably needs to be in 0.21?

**Comment by zmerlynn** [comment]
LGTM.

Can I get a lead for ok-to-merge ASAP? I'm going to cut as soon as I can get #10839 in.

File: cluster/ubuntu/skydns-rc.yaml.template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 kind: ReplicationController
 apiVersion: v1beta1
 id: kube-dns
-namespace: default
+namespace: kube-system
```
Comment (bgrant0607) [inline cluster/ubuntu/skydns-rc.yaml.template]: This doesn't do any harm, but these v1beta1 yaml files are completely broken at this point.

**Comment by bgrant0607** [comment]
ok to merge

None of these changes are critical for 0.21, though.

cluster/ubuntu is completely broken (still), since it uses v1beta1.

**Comment by zmerlynn** [comment]
cc @yujuhong for merge

**Comment by alex-mohr** [comment]
@bgrant0607 It's the fluentd change, not the ubuntu change, that we need to for 0.21?

**Comment by a-robinson** [comment]
It's the libvirt-coreos stuff, if anything. The fluentd change definitely isn't needed for 0.21

**Comment by satnam6502** [comment]
Is it worth us fixing the v1beta{1,2,3{ specs or shall we leave this to @lhuard1A @WIZARD-CXY ?

**Comment by zmerlynn** [comment]
`libvirt-coreos` looks correct, so I think it's just @WIZARD-CXY.

**Comment by WIZARD-CXY** [comment]
@zmerlynn I have already update the dns to api v1 , see this pr. I just wait too long for @erictune a green light.

**Pull request merged**